fixtures/tomshine/3-led-par-light-rgbuv.json

```diff
@@ -30,38 +30,74 @@
     }
   },
   "availableChannels": {
-    "Special Function": {
+    "No Function 1": {
+      "name": "No Function",
+      "capability": {
+        "type": "NoFunction"
+      }
+    },
+    "No Function 2": {
+      "name": "No Function",
+      "capability": {
+        "type": "NoFunction"
+      }
+    },
+    "Special Functions": {
       "defaultValue": 0,
       "capabilities": [
         {
           "dmxRange": [0, 10],
-          "type": "NoFunction"
+          "type": "NoFunction",
+          "switchChannels": {
+            "No Function / Color Presets": "No Function 1",
```

Comment:
Given that the "Color Presets" channel is switched to in all other capabilities, do you think it makes sense to make this a switched channel at all? I'd rather hard-code the "Color Presets" in the mode to simplify the logic.

Reply:
Makes sense, I hardcoded it!